cg/meta/demultiplex/status_db_storage_functions.py

```diff
@@ -36,6 +37,7 @@ def store_flow_cell_data_in_status_db(
         LOG.info(f"Flow cell added to status db: {parsed_flow_cell.id}.")
     else:
         LOG.info(f"Flow cell already exists in status db: {parsed_flow_cell.id}.")
+        flow_cell.status = FlowCellStatus.ON_DISK
```

Comment:
does this need to be committed?

Reply:
Yes! Luckily that's already done below 👍